src/tof/chopper.py

```diff
@@ -69,26 +74,49 @@ def open_close_times(
         ----------
         time_limit:
             Determines how many rotations the chopper needs to perform to reach the time
-            limit.
+            limit. If not specified, the chopper will perform a single rotation.
         unit:
             The unit of the returned times. If not specified, the unit of `time_limit`
             is used.
         """
+        if time_limit is None:
+            time_limit = sc.scalar(0.0, unit='us')
         if unit is None:
             unit = time_limit.unit
         nrot = max(int(sc.ceil((time_limit * self.frequency).to(unit='')).value), 1)
         # Start at -1 to catch early openings in case the phase or opening angles are
         # large
         phases = sc.arange(uuid.uuid4().hex, -1, nrot) * two_pi + self.phase.to(
             unit='rad'
-        )
+        ) * (int(self.direction == 'clockwise') * 2 - 1)
```

Comment:
Like this, any typos in `direction` will mean "anticlockwise". How about either validating the direction, or using a dict:

```suggestion
        ) * {'clockwise': 1, 'anticlockwise': -1}[self.direction]
```

Reply:
Yeah I wasn't sure about the naming. I don't know if `anticlockwise` is more intuitive than `counter-clockwise`.
Also, is it `anticlockwise` or `anti-clockwise`?

The best would probably to use an enum, but I don't know if that makes it impractical to use?
```Py
chopper = tof.Chopper(
    frequency=14*Hz,
    direction=tof.AntiClockwise,
    ...
    )
```

Or maybe we should have two classes? `ChopperClockwise` and `ChopperAnticlockwise`?
A bit verbose but hard to get wrong?

Reply:
In the end, I used
```Py
Clockwise = Literal["clockwise"]
AntiClockwise = Literal["anticlockwise"]
```
to define the two options. It seems to work fine, but I am not sure if there is anything bad/dangerous about doing it like that.